resources/transactions.ts

```diff
@@ -1,7 +1,7 @@
 import { BaseListParams, Include, Meta, Sort, Tags, UnitConfig, UnitResponse } from "../types/common"
 import { Customer } from "../types/customer"
 import { Account } from "../types/account"
-import { PatchTransactionRequest, Transaction } from "../types/transactions"
+import { PatchBookOrChargebackRequest, PatchTransactionRequest, Transaction } from "../types/transactions"
```

Comment:
let's rename PatchBookOrChargebackRequest to PatchTransactionWithRelationshipsRequest

```suggestion
import { PatchTransactionWithRelationshipsRequest, PatchTransactionRequest, Transaction } from "../types/transactions"
```

Reply:
Done